src/estimagic/visualization/profile_plot.py

```diff
@@ -160,13 +160,12 @@ def create_solution_times(df, runtime_measure, converged_info, return_tidy=True)
             problem, algorithm and runtime_measure. The values are either the number
             of evaluations or the walltime each algorithm needed to achieve the
             desired precision. If the desired precision was not achieved the value is
-            set to np.inf (for n_evaluations) or 7000 days (for walltime since there
-            no infinite value is allowed).
+            set to np.inf.
 
     """
     solution_times = df.groupby(["problem", "algorithm"])[runtime_measure].max()
-    solution_times = solution_times.unstack()
-    solution_times[~converged_info] = np.inf
+    solution_times = solution_times.unstack().astype(float)
```

Comment:
So what is it if not a float already? And does the unit matter or do we just not care here?

Reply:
It is an int. For the runtime measures `n_batches` and `n_evaluations` an integer makes more sense. However, I do not think that converting to a float will change any subsequent calculations. But I haven't written this code, so I am not entirely sure.

Reply:
Ah, then I would just put it in the line below, makes it clearer that it is done so that inf is supported.